ocaml/libs/xapi-rrd/lib/rrd.ml

```diff
@@ -968,9 +981,7 @@ let xml_to_output rrd output =
     (fun output ->
       tag "version" (data "0003") output ;
       tag "step" (data (Int64.to_string rrd.timestep)) output ;
-      tag "lastupdate"
-        (data (Printf.sprintf "%Ld" (Int64.of_float rrd.last_updated)))
-        output ;
+      tag "lastupdate" (data (Utils.f_to_s rrd.last_updated)) output ;
```

Comment:
I don't think this change is explained in the commit message, could you add that this adds decimals to the output format (and this is why the external interface needs to be maintained, which is mentioned in the commit message)